epub33/rs/index.html

```diff
@@ -1604,23 +1604,46 @@ <h4>Initial containing block dimensions</h4>
 									data-tests="#lay-fxl-xhtml-icb,#lay-fxl-xhtml-icb_multi">They MUST clip content
 									positioned outside of the ICB.</span>
 							</p>
+
 							<p id="confreq-fxl-rs-xhtml-icb-units" data-tests="#lay-fxl-xhtml-icb_units">If the width or
 								height values in the <code>viewport meta</code> tag contain a non-numeric character but
 								start with a number (e.g., the value includes a unit of length declaration such as
 								"500px"), the number prefix SHOULD be used as the pixel value, otherwise the value MUST
 								be treated as invalid.</p>
-							<p>If the <code>viewport meta</code> does not contain a width or a height value, or if these
-								values are invalid, reading systems SHOULD consider these as having the values
-									<code>device-width</code> and <code>device-height</code>, respectively.</p>
+
+							<p>
+								If the <code>viewport meta</code> tag is invalid, the reading system MAY attempt to
+								extract valid <code>width</code> and <code>height</code> values and, if successful, use those to establish
+								the initial containing block.
+							</p>
+							
+							<p>
+								If the <code>viewport meta</code> does not contain a width or a height value, or if these
+								values are invalid, reading systems MAY choose to:
```

Comment:
Do we want to place the MAY here on the list of suggestions, which then makes them the normative list of what you're allowed to do?

```suggestion
								values are invalid, reading systems MAY supply the values. For example, a reading system might:
```